src/lxc/conf.c

```diff
@@ -355,10 +355,11 @@ static int run_script_argv(const char *name, const char *section,
 
 	size += strlen(hook) + 1;
 
+	size += strlen("exec");
```

Comment:
That escaped me that should be a static `sizeof("exec")`. Fixing this in #2026 since I need to touch that code anyway. :)

Reply:
If you want to use `sizeof` instead, it should be `sizeof("exec") - 1`. The compiler should be smart enough to do the right thing though

Reply:
> If you want to use sizeof instead, it should be sizeof("exec") - 1.

No, not really I'm just leaveing it as `sizeof("exec")` since this automatically accounts for the added `' '`. In my branch I'm doing a `size++` after each `strlen()`. In this case I'm just going to leave the `sizeof()` and not do a `size++`. But feel free to comment on the #2026. :)

> The compiler should be smart enough to do the right thing though

Yeah, I agree but I like the code to reflect that this is information that is available statically.

Reply:
Right, cleaner indeed ;)